send-trace-to-jaeger.mjs

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs'
+import eventStream from 'event-stream'
+import retry from 'async-retry'
+import fetch from 'node-fetch'
+
+const file = fs.createReadStream(process.argv[2])
+
+const localEndpoint = {
+  serviceName: 'nextjs',
+  ipv4: '127.0.0.1',
+  port: 9411,
+}
+
+// Jaeger supports Zipkin's reporting API
+const zipkinUrl = `http://${localEndpoint.ipv4}:${localEndpoint.port}`
+const jaegerWebUiUrl = `http://${localEndpoint.ipv4}:16686`
+const zipkinAPI = `${zipkinUrl}/api/v2/spans`
```

Comment:
Why is there a zipkin url here in a file named `send-trace-to-jaeger.mjs`?

Reply:
Jaeger supports the Zipkin format with the same api endpoint